systest/build.rs

```diff
@@ -22,7 +22,9 @@ fn main() {
     });
     cfg.skip_field(|struct_, f| {
         // this field is marked as const which ctest complains about
-        struct_ == "git_rebase_operation" && f == "id"
+        (struct_ == "git_rebase_operation" && f == "id") ||
+        // the real name of this field is ref but that is a reserved keyword
+        (struct_ == "git_worktree_add_options" && f == "reference")
```

Comment:
Could `r#ref` be used as a field name instead?

Reply:
ctest depends on old syntex_syntax2, which doesn't know about raw identifier. It complaints that "found invalid character; only `#` is allowed in raw string delimitation"

Reply:
Ah, that's ok then.